psi4/driver/schema_wrapper.py

```diff
@@ -545,14 +545,8 @@ def run_json_qcschema(json_data, clean, json_serialization, keep_wfn=False):
 
     # Handle special properties case
     if json_data["driver"] == "properties":
-        if "properties" in json_data["model"]:
-            kwargs["properties"] = [x.lower() for x in json_data["model"]["properties"]]
-
-            extra = set(kwargs["properties"]) - can_do_properties_
-            if len(extra):
-                raise KeyError("Did not understand property key %s." % kwargs["properties"])
-        else:
-            kwargs["properties"] = list(can_do_properties_)
+        if not "properties" in kwargs:
+            kwargs["properties"] = list(default_properties_)
```

Comment:
What happens if the asked-for property cannot be computed for whatever reason (wrong keyword as well as incompatible method and property)? Do we abort before any calculations are done, or only when we try to compute the properties?

Reply:
good question, it sort of depends on the method. for cc props,`proc.py` will print `"The following properties are not currently supported: ['PROPERTY']"`. any valid properties in the list are still computed. for scf, it prints `"SCF Property: Feature 'PROPERTY' is not recognized."` and suggests alternatives if possible, but no properties are computed, even the valid ones in the list. CI goes through the `proc_utils.oeprop_validator()`, algebraic diagrammatic construction does something similar to scf, and I don't think DFMP2 checks until somewhere later down the line.  

so as far as the Schema (and whatever is using it, like QCEng) is concerned, I guess the answer is "only when we try to compute the properties," then it's up to `proc.py` to have its own sanity checks depending on the module. I think it generally does a good job, so I think another layer of validating the method/prop combo in the schema wrapper isn't necessary.

Reply:
OK, I am convinced that `schema_wrapper` may not be the best place to validate those inputs. 

I do think we need to make sure to "not crash" if it turns out a requested property (via qcschema or other spec) is not supported or is a typo. I want to avoid a situation where one requests a property that may not exist for a particular method, and then **nothing** gets returned instead of at least the stuff that has been computed already.